include/envoy/server/options.h

```diff
@@ -65,6 +65,11 @@ class Options {
   virtual const std::string& configPath() PURE;
 
   /**
+   * @return const std::string& the path to the v2 bootstrap file.
+   */
+  virtual const std::string& bootstrapPath() PURE;
```

Comment:
Q: Is it worth having a different option for this? I feel like `-c` could just detect whether it's JSON or proto (or v2 JSON) via attempting parsing?